README.md

```diff
@@ -1,2 +1,36 @@
-# dst-parser
-This repository is managed by Terraform!
+# Example docstring parser
```

Comment:
```suggestion
# Docstring Parser
```

Reply:
I am not as sure about this part, the parser is intended specifically for our example code and it may look more vague, but I'll add that and rename it if I have a better name later.

Reply:
you can say which code base and documentation it was designed for, it can be "docstring parser for Iroha tutorial"

Reply:
anyway my main point is that it's not an "example" :) we can sure come up with a better name for this but just doctring parser should also be ok, and adding a paragraph on what this was created for will help clear up any confusion